Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed 3D Vector and 1D Axis binding dropdown usage in Input Actions Editor throwing NotImplementedExceptions.
 - Fixed several missing tooltips from the Action/Binding Properties pane in Input Actions Editor.
 - Fixed an issue in the InputActionAsset Editor where ControlType wasn't updated when ActionType changed.
+- Fixed an issue where newly created action map names were not editable
```

Comment:
Minor: Missing full stop at end.

Reply:
There's always one tiny thing lol!

I had to merge the CHANGELOG.md anyway - I've added a full stop!